docs/releasenotes.md

```diff
@@ -5,6 +5,28 @@ title: Releasenotes
 
 # Release notes BAG Huidige bevragingen API
 
+## Versie 1.2.0 (22-06-2021)
+In deze release zijn enkele bugs in productie opgelost. Tevens zijn er enkele wijzigingen aan de API-specificaties en de feature-files aangebracht.
+Er worden nu adresregels geleverd met daarin het samengestelde adres.
+De Header Content-Crs is voorzien van een default waarde.
+Op /adressen worden de resultaten gesorteerd geretourneerd.  
+
+### Opgeloste issues
+
+  | Link Github	| Beschrijving |
+  | --- | --- |
+  | [234](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/234) | gebruik relatieve urls voor interne links |
+  | [293](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/293) | Expand gerelateerde resource als relatie in onderzoek is |
+  | [294](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/294) | Als gebruiker (nader te bepalen) wil ik de inhoud van de mogelijkOnjuist gegevensgroep kunnen filteren |
+  | [309](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/309) | wijzigen description zoekparameter zoekresultaat get/adressen/zoek e.a. |
+  | [338](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/338) | Underscores verwijderen uit componentnamen en enumeratienamen. |
+  | [343](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/343) | Gebruiker mag niet worden verplicht om Accept-Crs te definieren |\
+  | [360](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/360) | Als gebruiker wil ik het samengesteld adres krijgen |  
+  | [363](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/363) | als developer van de BRK API wil ik de korteNaam kunnen leveren bij het adres van een kadastraal object |  
+  | [364](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/364) | toevoegen controle op minimale en maximale lengte invoer zoek |  
```

Comment:
Ik denk dat het handig is om dit beter te beschrijven. Hier staat dat er controle is toegevoegd, dus ik verwacht dat minLength en maxLength is toegevoegd aan de zoek parameter. Maar die kom ik daar niet tegen. Als ik de issue lees, concludeer ik dat de gekozen oplossing is: geen zoek resultaat bij een zoek waarde gelijk aan 0 of groter dan 255. Titel van issue komt niet overeen met gekozen oplossing.